src/vs/workbench/services/editor/browser/editorResolverService.ts

```diff
@@ -333,12 +342,19 @@ export class EditorResolverService extends Disposable implements IEditorResolver
 		});
 	}
 
-	public getEditorIds(resource: URI): string[] {
-		const editors = this.findMatchingEditors(resource);
-		if (editors.find(e => e.editorInfo.priority === RegisteredEditorPriority.exclusive)) {
-			return [];
+	public getEditors(resource?: URI): RegisteredEditorInfo[] {
```

Comment:
@lramos15 fyi I changed this method to allow me to return all registered editors.

Reply:
Is there a reason we now have `getEditors` and `getEditorIds`?